HomeWork10/HomeWork10/Services/InstrumentsService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using HomeWork10.Entities;
+using HomeWork10.Models;
+using HomeWork10.Models.Enums;
+using HomeWork10.Repositories.Abstructs;
+using HomeWork10.Services.Abstractions;
+using System.Drawing;
+
+namespace HomeWork10.Services
+{
+    internal class InstrumentsService : IInstrumentsService
+    {
+        private readonly ILoggerService _logger;
+        private readonly IInstrumentsRepository _repository;
+        private readonly INotifyService _notify;
+        public InstrumentsService(ILoggerService loggerService, IInstrumentsRepository instrumentsRepository, INotifyService notifyService)
+        { 
+            _logger = loggerService;
+            _notify = notifyService;
+            _repository = instrumentsRepository;
+        }
+        public string AddInstruments(string name, int consumes, Color color, TypeInstruments typeAppliance, int weight)
+        {
+            BrendEntity brend = new BrendEntity()
+            {
+                Name = "Colosok",
+                Country = "Ukraine",
+                BrandRegistration = "Italy"
+            };
```

Comment:
I think it will be better to create brend repository and return some brends from there

Reply:
i don`t know where realithations repository. in service or repository